runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -395,6 +400,7 @@ public static CorfuRuntimeParametersBuilder builder() {
             private int holeFillRetry = 10;
             private Duration holeFillRetryThreshold = Duration.ofSeconds(1L);
             private Duration holeFillTimeout = Duration.ofSeconds(10);
+            private boolean cacheEntryMetricsDisabled = false;
```

Comment:
Set to true? otherwise if CorfuRuntimeParametersBuilder is used without explicitly setting this parameter it will enable this metrics.

Reply:
ahh yes, my bad. I wanted it to test out the metrics so had changed it to "false". Did not realize I pushed after setting it to false.
Thanks for catching this!